go-controller/pkg/util/dns.go

```diff
@@ -8,16 +8,18 @@ import (
 
 	"github.com/miekg/dns"
 
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/klog/v2"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 )
 
 const (
-	// defaultTTL is used if an invalid or zero TTL is provided.
+	// defaultTTL is used when the resolution fails
 	defaultTTL = 30 * time.Minute
+
+	// minimalTTL is used when a 0 TTL is answered
+	minimalTTL = 10 * time.Second
```

Comment:
the egressfirewall dns resolution is best effort but maybe miniamalTTL should be 1 second. @dcbw do you think that would add too much network traffic